api/query_test.go

```diff
@@ -367,7 +367,7 @@ func TestFenceErrorHTTPResponse(t *testing.T) {
 			fenceQuery: "{\"vds\":\"" + well_known +
 				"\", \"coordinate_system\":\"ilxl\", \"sas\": \"n/a\"}",
```

Comment:
It is not really important (in theory all parameters can be removed altogether, as we just check for non-existence of one particular), but maybe update coordinate_system here and in neighbor test?